ui/app/components/gas-customization/gas-modal-page-container/index.scss

```diff
@@ -0,0 +1,50 @@
+.gas-modal-content {
```

Comment:
Should we stick with BEM-ish notation? Nesting classes makes styles harder to override when needed.